examples/csharp/Dockerfile

```diff
@@ -25,9 +25,8 @@ RUN rm -rf test/Twilio.Test/*/
 COPY unit-test/Rest test/Twilio.Test/
 
 RUN dotnet restore
-RUN dotnet build -c Release
-
-CMD ["/bin/bash", "-c", "dotnet test test/Twilio.Test/Twilio.Test.csproj --no-build | tee /local/test-report.out"]
+RUN	dotnet build Twilio.sln
+CMD ["/bin/bash", "-c", "dotnet test test/Twilio.Test/Twilio.Test.csproj --no-build"]
```

Comment:
Why this change? The benefit of putting the output in a `/local` file was to see the results without needing to connect to the container.